oat-sa/extension-tao-testqti
# Feature/ADF-1789/Side by side authoring

### Related to: https://oat-sa.atlassian.net/browse/ADF-1789

Requires: 
 - [x] https://github.com/oat-sa/tao-core/pull/4119

### Summary

Add the side-by-side UI in the test authoring when translating.

![image](https://github.com/user-attachments/assets/74e80781-fca3-4664-9c88-4c68d68b14f0)

### Details

Add support for:
- opening a test for translation: side-by-side UI

### How to test
- checkout the branch: `git checkout -t origin/feature/ADF-1789/side-by-side-authoring`
- also make sure to checkout the companion PRs (`feat/HKD-6/integration`)
- make sure to have the Feature Flags activated
```
FEATURE_FLAG_TRANSLATION_ENABLED=true
FEATURE_FLAG_UNIQUE_NUMERIC_QTI_IDENTIFIER=true
```
- make sure to have the `Test Translator` role assigned to your user
- have a test ready for translation and add/edit a translation: it should present the side-by-side authoring
- from a regular test, also open the authoring: it should present the usual authoring
